tests/node_decommission/node_decommission_test.go

```diff
@@ -0,0 +1,104 @@
+package tests
+
+import (
+	"fmt"
+	"math/rand"
+	"testing"
+	"time"
+
+	"github.com/libopenstorage/openstorage/api"
+	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/reporters"
+	. "github.com/onsi/gomega"
+	"github.com/portworx/sched-ops/task"
+	"github.com/portworx/torpedo/drivers/node"
+	"github.com/portworx/torpedo/drivers/scheduler"
+	. "github.com/portworx/torpedo/tests"
+)
+
+const (
+	defaultTimeout       = 6 * time.Minute
+	defaultRetryInterval = 10 * time.Second
+)
+
+func TestDecommissionNode(t *testing.T) {
+	RegisterFailHandler(Fail)
+
+	var specReporters []Reporter
+	junitReporter := reporters.NewJUnitReporter("/testresults/junit_DecommissionNode.xml")
+	specReporters = append(specReporters, junitReporter)
+	RunSpecsWithDefaultAndCustomReporters(t, "Torpedo: DecommissionNode", specReporters)
+}
+
+var _ = BeforeSuite(func() {
+	InitInstance()
+})
+
+var _ = Describe("{DecommissionNode}", func() {
+	testName := "decommissionnode"
+	It("has to decommission a node and check if node was decommissioned successfully", func() {
+		var contexts []*scheduler.Context
+		for i := 0; i < Inst().ScaleFactor; i++ {
+			contexts = append(contexts, ScheduleAndValidate(fmt.Sprintf("%s-%d", testName, i))...)
+		}
+
+		Step("pick a random node and decommission it", func() {
```

Comment:
add a for loop here and do the entire workflow === chaos level.

So if chaos level is given as 5, you will decommission 5 nodes (one at a time).

So 

loop 1
- prepare worker1 for decommission
- decommission worker1
- rejoin worker1

loop 2
- prepare worker4 for decommission
- decommission worker4
- rejoin worker4

... so on